polyfill/intersection-observer.js

```diff
@@ -147,6 +161,45 @@ IntersectionObserver.prototype.POLL_INTERVAL = null;
 IntersectionObserver.prototype.USE_MUTATION_OBSERVER = true;
 
 
+/**
+ * Set up the polyfill in the cross-origin mode. The result is the
+ * updater function that accepts two arguments: `boundingClientRect` and
+ * `intersectionRect` - just as these fields would be available to the
+ * parent via `IntersectionObserverEntry`. This function should be called
+ * each time the iframe receives intersection information from the parent
+ * window, e.g. via messaging.
+ * @return {function(DOMRect, DOMRect)}
+ */
+IntersectionObserver.polyfillSetupCrossOriginUpdater = function() {
```

Comment:
I'm not crazy about the use of the `polyfill` prefix here. I know you're trying to avoid collisions with the native API, but this kinda makes it sound like the method is polyfilling something.

How would you feel about `_setupCrossOriginUpdater()` or `__setupCrossOriginUpdater()`?

(BTW, I know I'm already kinda breaking this with `POLL_INTERVAL` etc., but those feel a bit safer since they're all caps. Arguably they should also be `__POLL_INTERVAL` though...)

Reply:
I'd like that, but I'm a little freaked out by the possibility that some compilers/bundles will consider it a module-boundary-private and obfuscate. However, the intent of this function is to be called outside of this module. I run into such things occasionally and so rather careful about cross-module function names with a head/tail underscore.

Reply:
Renamed to `_setupCrossOriginUpdater`.